#!/usr/bin/env python
PACKAGE = "depth_colorizer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_distance", double_t, 0, "min_distance parameter", 500, 0, 10000)
gen.add("max_distance", double_t, 0, "max_distance parameter", 1500, 0, 10000)

exit(gen.generate(PACKAGE, "depth_colorizer", "DynamicParams"))

